File: server/src/main/java/com/talkeasy/server/dto/user/LastChatResponse.java
package com.talkeasy.server.dto.user;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
@AllArgsConstructor
public class LastChatResponse {
    private String msg; // 메시지 내용
    private String created_dt; // 생성 시간?
    private Integer readCnt; // 읽음 수정
}

File: TalkEasy/core/data/src/main/java/com/ssafy/talkeasy/core/data/remote/datasource/follow/LastChatResponse.kt
package com.ssafy.talkeasy.core.data.remote.datasource.follow

import com.google.gson.annotations.SerializedName
import com.ssafy.talkeasy.core.data.remote.datasource.base.DataToDomainMapper
import com.ssafy.talkeasy.core.domain.entity.response.LastChat

data class LastChatResponse(
    @SerializedName("msg")
    val msg: String,
    @SerializedName("created_dt")
    val created_dt: String,
    @SerializedName("readCnt")
    val readCnt: Int,
) : DataToDomainMapper<LastChat> {

    override fun toDomainModel(): LastChat =
        LastChat(message = msg, time = created_dt, readCount = readCnt)
}
